platform/security/pdp/security-pdp-authzrealm/src/main/java/ddf/security/pdp/realm/SimpleAuthzRealm.java

```diff
@@ -85,15 +83,8 @@
      */
     public static final String ACCESS_ROLE_LIST = "accessRoleList";
 
-    /**
-     * Identifies the key used to retrieve a List of Strings that represent actions that are open
-     * for any role to access. Each string is the action corresponding to the SOAP action presented
-     * in the SOAP request. If the specified string is contained in the SOAP action string, then
-     * access is automatically allowed for this operation.
-     */
-    public static final String OPEN_ACCESS_ACTION_LIST = "openAccessActionList";
-
-    private static final XLogger LOGGER = new XLogger(LoggerFactory.getLogger(SimpleAuthzRealm.class));
+    private static final XLogger LOGGER = new XLogger(
```

Comment:
I don't want to perpetuate the use of XLogger. If you see it is being used, rip it out and replace it with the standard slf4j one.